common/index.ts

```diff
@@ -17,3 +17,9 @@ export const PLUGIN_ID = 'opendistroSecurity';
 export const PLUGIN_NAME = 'opendistro_security';
 
 export const API_PREFIX = '/api/v1/opendistro_security';
+
+export const OPENDISTRO_URL_PREFIX = 'opendistro_security#/';
```

Comment:
The `opendistro_security ` comes from here: https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/blob/dev/public/plugin.ts#L36
Suggest change both places to `PLUGIN_NAME`.

Reply:
Sure